document/core/syntax/modules.rst

```diff
@@ -112,16 +120,17 @@ Conventions
    For example, if :math:`\instr^\ast` is :math:`(\DATADROP~x) (\MEMORYINIT~y)`, then :math:`\freedataidx(\instr^\ast) = \{x, y\}`, or equivalently, the vector :math:`x~y`.
 
 
-.. index:: ! type definition, type index, function type
+.. index:: ! type definition, type index, function type, event type
    pair: abstract syntax; type definition
 .. _syntax-typedef:
 
 Types
 ~~~~~
 
-The |MTYPES| component of a module defines a vector of :ref:`function types <syntax-functype>`.
+The |MTYPES| component of a module defines a vector of :ref:`function types <syntax-functype>`,
+which may also be part of an :ref:`event type <syntax-eventtype>`.
```

Comment:
```suggestion
The |MTYPES| component of a module defines a vector of :ref:`function types <syntax-functype>`.
```
Function types have a number of uses, e.g., they can also describe block types or the types of instructions. So I wouldn't bother mentioning event types specifically at this point.